default/python/AI/freeorion_tools/bonus_calculation.py

```diff
@@ -0,0 +1,12 @@
+from typing import NamedTuple
+
+
+class Bonus(NamedTuple):
+    available: bool
+    min_stability: float
+    value: float
+
+    def get_bonus(self, stability: float) -> float:
+        if not self.available:
+            return 0.0
+        return self.value if stability > self.min_stability else 0.0
```

Comment:
```suggestion
        if not self.available or stability < self.min_stability:
            return 0.0
        return self.value
```

Reply:
There is a simple rule, expressions with `not` are more challenging for humans to read.

 `not self.available or stability < self.min_stability` -> `self.available or stability >= self.min_stability`